CMakeLists.txt

```diff
@@ -90,6 +92,11 @@ set (P4C_CXX_FLAGS "")
 set (P4C_LIB_DEPS)
 
 
+# Support a legacy option.
+if (ENABLE_UNIFIED_COMPILATION)
```

Comment:
Feel free to emit an warning here that this option is deprecated and it will be removed after some time.